src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -118,58 +118,58 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AvoidAsyncVoidTitle" xml:space="preserve">
-    <value>Avoid Async Void</value>
+    <value>Avoid 'async void'.</value>
```

Comment:
There shouldn't be a period for title.

Reply:
The analyzer should have caught that, is it broken?

Reply:
@Evangelink Good catch. The reason for that is simply this is a dead resource. It's not used currently. Probably the relevant analyzer was deleted at some point, and it was forgotten to remove these resources.

Reply:
There are lots of dead resources. Let me cleanup this first in a separate PR.